tests/lib/ocpp/v16/json/TxProfile_02.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
I think it would be helpful to other devs to rename these JSON files or add a `README.md` that explains what each one is. (Something like "A daily recurring `TxProfile` beginning on 2024-01-17 that limits charging to 1-phase and 2 kW for 18-minutes beginning at 0400." for this profile? Or a table with descriptions like this? I'm open to suggestions.)

Reply:
Your wish is granted.